### PR TITLE
centos7 dockerfile

### PR DESCRIPTION
**Pull request by raticate**

### CODE REVIEW

File: tchsm/lib/centos7/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+            libedit-devel \
+            libuuid-devel \
+            make \
+            m4 \
+            sqlite-devel \
+            wget
+
+RUN yum install -y gcc gcc-c++
+
+# RUN curl -L https://people.centos.org/tru/devtools-2/devtools-2.repo -o /etc/yum.repos.d/devtools-2.repo && yum install -y devtoolset-2-gcc devtoolset-2-binutils devtoolset-2-gcc-c++
```
Comment (fmontoto) [inline tchsm/lib/centos7/Dockerfile]: remove this?

File: tchsm/lib/centos7/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+ENV PKG_CONFIG_PATH ${PKG_CONFIG_PATH}:/usr/local/lib/pkgconfig
+# ENV CXX /opt/rh/devtoolset-2/root/usr/bin/g++
```
Comment (fmontoto) [inline tchsm/lib/centos7/Dockerfile]: Remove?

**Review by fmontoto — CHANGES_REQUESTED** [review]

File: tchsm/lib/centos7/Dockerfile (hunk 1 above)
Comment (raticate) [reply tchsm/lib/centos7/Dockerfile]: it was necessary. I got "couldn't find compiler" errors.

File: tchsm/lib/centos7/Dockerfile (hunk 2 above)
Comment (raticate) [reply tchsm/lib/centos7/Dockerfile]: it was necessary for zeromq

**Review by fmontoto — APPROVED** [review]

**Pull request merged**